src/Symfony/Component/Console/Tests/EventListener/CommandColorListenerTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Symfony\Component\Console\Tests\EventListener;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\EventListener\CommandColorListener;
+use Symfony\Component\Console\Event\ConsoleCommandEvent;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CommandColorListenerTest extends TestCase
+{
+    /**
+     * @var CommandColorListener
+     */
+    protected $fixture = null;
+
+    /**
+     * @var string
+     */
+    protected static $varName = null;
+
+    public static function setUpBeforeClass()
+    {
+        do {
+            self::$varName = uniqid('CommandColorListener');
```

Comment:
do we really need to use uniqid here ?

Reply:
We don't need to.  Sorry, it is something from the project's standards I work on.